test/app/components/haml_component.html.haml

```diff
@@ -1,3 +1,8 @@
 %div
   = content
   = @message
+
+- if @url
+  %div
+    = helpers.link_to(@url) do
```

Comment:
```suggestion
    = link_to(@url) do
```

Using `helpers` here takes us to the view context of this template's controller. See the definition of [helpers in lib/view_component/base.rb](https://github.com/github/view_component/blob/c0129a93fbd9d7f37e0c85c82984a5d05b9f3fc1/lib/view_component/base.rb#L114-L117). It's unneeded for the `link_to` helper. It's already available in the template's context.